### PR TITLE
FUSETOOLS-2487 - Upgrade from Camel version from 2.19.1 to 2.19.2

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

File: core/plugins/org.fusesource.ide.camel.model.service.impl/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -12,17 +12,18 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.11.0",
  org.jboss.tools.foundation.core;bundle-version="1.2.0",
  org.jboss.tools.foundation.ui;bundle-version="1.2.0",
  org.fusesource.ide.foundation.core;bundle-version="10.0.0",
- org.fusesource.ide.camel.model.service.core;bundle-version="10.0.0"
+ org.fusesource.ide.camel.model.service.core;bundle-version="10.0.0",
+ org.apache.commons.httpclient;bundle-version="3.1.0"
```
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.impl/META-INF/MANIFEST.MF]: is that lib a new requirement?

Comment (apupier) [inline core/plugins/org.fusesource.ide.camel.model.service.impl/META-INF/MANIFEST.MF]: I don't remember adding this bundle requirement (and so also why I added it)

Comment (apupier) [reply core/plugins/org.fusesource.ide.camel.model.service.impl/META-INF/MANIFEST.MF]: I created https://github.com/jbosstools/jbosstools-fuse/pull/1087 to remove it

Comment (apupier) [reply core/plugins/org.fusesource.ide.camel.model.service.impl/META-INF/MANIFEST.MF]: finally this wasn't added by error, now that tests are failing on the other PR, i remember now that it was needed

**Pull request merged**